mytemplates/cors/corssouf.yaml

```
@@ -0,0 +1,45 @@
+  description: Detects Cross-Origin Resource Sharing (CORS) misconfigurations that allow arbitrary origins and credentials, enabling unauthorized data access or theft.
+  reference:
+    - https://owasp.org/www-project-secure-headers/#cross-origin-resource-sharing
+    - https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+  tags: cors, misconfiguration, credentials, data-leakage, vulnerability
+
+http:
+  - method: GET
+    path:
+      - "{{BaseURL}}"
+    headers:
+      Origin: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)